### PR TITLE
[Hotfix] Fix kRO packets

### LINKED ISSUE #3389: ShieldRO (RagexeRE_2018_04_18a) - Disconnect after public chat (c <chat>)
## ------------------ Openkore Issues Template ------------------

* **Openkore version git**: Latest

* **Server**: **iRO**

* **Bug Report / Feature Request**: 

* **Summary**: disconnected after trying to chat in public.
*   [
image
](url)

### PR DESCRIPTION
**Pull request by alisonrag**
packets are overrided in other servertypes, so we need to redefine these packets again

this fixes #3389

### CODE REVIEW

**Comment by ya4ept** [comment]
we broke it here:
https://github.com/OpenKore/openkore/pull/3382/files#diff-13b82aa4bf1879a3a856f7000f11cdf314f847768adcd08c152cf1c2e05f0868

**Review by ya4ept — APPROVED** [review]

File: src/Network/Send/kRO/RagexeRE_2018_04_04b.pm [hunk 1]
```diff
@@ -18,7 +18,34 @@ use base qw(Network::Send::kRO::RagexeRE_2018_03_07b);
 sub new {
 	my ($class) = @_;
 	my $self = $class->SUPER::new(@_);
-	
+
```
Comment (sctnightcore) [inline src/Network/Send/kRO/RagexeRE_2018_04_04b.pm:21]: i have no idea with it .

Comment (alisonrag) [reply src/Network/Send/kRO/RagexeRE_2018_04_04b.pm:21]: shuffle packets stop at this servertype, so we need to redefine these packets again

**Comment by alisonrag** [comment]
@ya4ept @sctnightcore do you want to take a look again before merge it?

**Comment by ya4ept** [comment]
have you tried to get to the char server? I get an error
```
[CircleRagnarok]
ip 103.52.146.53
port 6900
master_version 12
version 55
serverType kRO_RagexeRE_2020_03_04a
serverEncoding Western
gameGuard 0
addTableFolders kRO/RagexeRE_2020_07_09a;iRO/official;iRO
charBlockSize 155
pinCode 1
charDeleteDateType 1
rankingSystemType 1
private 1
```

**Comment by ya4ept** [comment]
new settings (OTT_ip and OTT_port):
```
[CircleRagnarok]
ip 103.52.146.53
port 6900
OTT_ip 103.52.146.53
OTT_port 6900
master_version 12
version 55
serverType kRO_RagexeRE_2020_03_04a
serverEncoding Western
gameGuard 0
addTableFolders kRO/RagexeRE_2020_07_09a;iRO/official;iRO
charBlockSize 155
pinCode 1
charDeleteDateType 1
rankingSystemType 1
private 1
```

**Comment by alisonrag** [comment]
after your last commit i can login without issues 
image

**Pull request merged**